test/runtests.jl

```diff
@@ -91,7 +93,9 @@ try
     NEW = GZip.GZLIB_VERSION > "1.2.3.9"
     pos = position(gzfile)
     NEW && (pos2 = position(gzfile,true))
-    @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
+    try 
+        @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
```

Comment:
It used to be in a large try block from line 29/31. Line 97 actually throws an error; the test checks that it throws the correct error. Without a try, the code execution would halt. Correct me if I'm wrong.